webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
Although I just spoke with our DOM team and they're saying that "In practice, web algorithms are treated as closing over everything they might care about that they don't get from somewhere explicitly".